### PR TITLE
Clarify whats experimental in 4.2

### PR DESCRIPTION
**Pull request by ibgreen**
doc updates.

### CODE REVIEW

File: docs/roadmap.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# Roadmap
```
Comment (1chandu) [inline docs/roadmap.md]: 👍 very clear and concise summary, I like it.

File: docs/upgrade-guide.md [hunk 2]
```diff
@@ -2,46 +2,55 @@
...
 | ---                   | ---               | ---     |
-| `useDevicePixelRatio` | `useDevicePixels` | |
+| `useDevicePixelRatio` | `useDevicePixels` | Changed due to feedback that the original name was confusing. |
 
 
-### Picking uniforms
+### Picking Uniforms
 
-`renderPickingBuffer` and `selectedPickingColor` are deprecated, and will be removed in next major version. It is recommended to use `luma.gl` picking module.
+The shader uniforms `renderPickingBuffer` and `selectedPickingColor` are deprecated, and will be removed in next major version. The props are still available but will generate a deprecation warning.
```
Comment (1chandu) [inline docs/upgrade-guide.md]: Actually these uniforms are always set and we don't generate deprecation warnings. We should ideally add deprecation warning if shader contains any of these uniforms, but we are not doing that now.

File: docs/whats-new.md [hunk 3]
```diff
@@ -1,53 +1,36 @@
...
 
 
-## Controller Classes
+## DeckGL: Viewport Transitions
 
-deck.gl's controller classes have been significantly refactored, providing you with lots of options for how to control how the user interacts with your viewports. Also the controllers are no longer categorized as "experimental" exports, so you can import and use them in your apps knowing that they will be supported in future releases the same way as any other deck.gl feature.
+A major new feature is support for Viewport Transitions.
 
-TBA...
+> TODO - Add more extensive description. This is our major official feature after all...
```
Comment (1chandu) [inline docs/whats-new.md]: I will add this section once viewportController/DeckGL merging is finalized. Thanks for `TODO` note.

File: docs/upgrade-guide.md (hunk 2 above)
Comment (ibgreen) [reply docs/upgrade-guide.md]: > Actually these uniforms are always set and we don't generate deprecation warnings. 

Good catch. Updated.

**Pull request merged**